app/models/scoreboard.rb

```diff
@@ -1,7 +1,10 @@
-class ScoreboardSetup < ActiveRecord::Base
```

Comment:
It won't.  I've already change the scoreboardHeader spec, so we'll need to update those calls, but only the 213 assignments and one other assignment use that, so we'll be fine.